website/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "website",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "tsc -b && vite build",
+    "dev": "vite",
+    "prepreview": "npm run build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@tsconfig/vite-react": "3.4.0",
+    "@types/react": "18.2.45",
+    "@types/react-dom": "18.2.18",
+    "@vitejs/plugin-react": "4.4.1",
+    "autoprefixer": "10.4.21",
+    "postcss": "8.5.3",
+    "tailwindcss": "3.4.14",
+    "typescript": "5.6.3",
+    "vite": "6.3.5",
+    "vite-tsconfig-paths": "5.1.4"
```

Comment:
Can dev deps be hoisted to the root of the repo? Just wondering...

Reply:
This are website related. They won't apply to other packages if any. The ones that are repo related (eslint, and others) are in the root.

The installation ends up hoisted in `node_modules` thanks to `npm workspaces`. Moving this to the outside will only add confusion imho